UnityProject/Assets/Scripts/Medical/DNAscanner.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Linq;
+using UnityEngine;
+using Mirror;
+
+public class DNAscanner : ClosetControl, ICheckedInteractable<MouseDrop>, IAPCPowered
```

Comment:
Codacy found an issue: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on 'cancelOccupiedAnim'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6387847)